targets/TARGET_Realtek/TARGET_AMEBA/TARGET_RTL8195A/device/TOOLCHAIN_GCC_ARM/rlx8195A-symbol-v02-img2.ld

```diff
@@ -0,0 +1,218 @@
+
+
+ENTRY(Reset_Handler)
+
+INCLUDE "mbed-os/targets/TARGET_Realtek/TARGET_AMEBA/TARGET_RTL8195A/device/TOOLCHAIN_GCC_ARM/export-rom_v02.txt"
```

Comment:
That can't be hardcoded like that. It will work for 'normal app builds' where mbed OS lives in mbed-os directory, but it will fail for any other arrangements (eg `mbed test`) @0xc0170 @c1728p9 do you guys know better way of doing it?